include/wx/uilocale.h

```diff
@@ -238,6 +249,21 @@ class WXDLLIMPEXP_BASE wxUILocale
     static wxString GetScriptAliasFromName(const wxString& scriptName);
     static wxString GetScriptNameFromAlias(const wxString& scriptAlias);
 
+    // These three methods are for internal use only.
```

Comment:
Do they still need to be public or can we move them to some private header?

Reply:
These methods could be moved to some private headers, of course. They only have to be available, wherever they are needed internally.

Reply:
The following static methods could be moved from class `wxUILocale` in `include/wx/uilocale.h` to class `wxUILocaleImpl` in `include/wx/private/uilocale.h`:
```c++
   static wxString GetScriptAliasFromName(const wxString& scriptName);
   static wxString GetScriptNameFromAlias(const wxString& scriptAlias);
   static wxString GetLikelySubtags(const wxString & fromTag);
   static int GetMatchDistance(const wxString& desired, const wxString& supported);
   static bool SameRegionGroup(const wxString& language, const wxString& desiredRegion, const wxString& supportedRegion);
```
@vadz Please confirm, if I should make this change.